gcc/config/aarch64/aarch64.cc

```diff
@@ -7055,6 +7110,90 @@ bitint_or_aggr_of_bitint_p (tree type)
   return false;
 }
 
+static int
+aarch64_arg_size (const function_arg_info &arg)
+{
+  HOST_WIDE_INT size;
+
+  /* Size in bytes, rounded to the nearest multiple of 8 bytes.  */
+  if (arg.type)
+    size = int_size_in_bytes (arg.type);
+  else
+    /* No frontends can create types with variable-sized modes, so we
+       shouldn't be asked to pass or return them.  */
+    size = GET_MODE_SIZE (arg.mode).to_constant ();
+
+  return ROUND_UP (size, UNITS_PER_WORD);
+}
+
+#if defined (TARGET_AARCH64_MS_ABI)
+
+/* Layout a function argument according to the AAPCS64 rules.  The rule
+   numbers refer to the rule numbers in the AAPCS64.  The Microsoft Arm64
+   variadic function call ABI uses only C.12-C15 rules.
+   See: https://learn.microsoft.com/en-us/cpp/build/arm64-windows-abi-conventions?view=msvc-170#addendum-variadic-functions */
+
+static void
+aarch64_ms_variadic_abi_layout_arg (cumulative_args_t pcum_v,
+				    const function_arg_info &arg)
+{
+  CUMULATIVE_ARGS *pcum = get_cumulative_args (pcum_v);
+  tree type = arg.type;
+  machine_mode mode = arg.mode;
+  int ncrn, nregs;
+  HOST_WIDE_INT size;
+
+  size = aarch64_arg_size (arg);
+  ncrn = pcum->aapcs_ncrn;
+  nregs = size / UNITS_PER_WORD;
+
+  if (ncrn < NUM_ARG_REGS)
+    {
+      /* The argument bytes are copied to the core registers.  */
+      if (nregs == 1 || GET_MODE_CLASS (mode) == MODE_INT)
+	{
+	  pcum->aapcs_reg = gen_rtx_REG (mode, R0_REGNUM + ncrn);
+	}
+      else
+	{
+	  rtx par;
+	  int i;
+
+	  /* Handle the case when argument is split between the last registers
+	     and the stack.  */
+	  if (ncrn + nregs > NUM_ARG_REGS) {
+	    pcum->aapcs_stack_words = ncrn + nregs - NUM_ARG_REGS;
+	    nregs -= pcum->aapcs_stack_words;
+	  }
+
+	  /* Generate load arg to registers intructions.  */
```

Comment:
```suggestion
	  /* Generate load arg to registers instructions.  */
```

Just a typo I noticed yesterday